### PR TITLE
Add option to toggle theme style (dark/light)

### PR DESCRIPTION
**Pull request by zetxek**
Implemented #113, using the theme toggler functionality from bootstrap 5.3 (https://github.com/search?q=repo%3Atwbs%2Fbootstrap%20bd-theme&type=code).

Video:
https://github.com/user-attachments/assets/8bc89ae3-6647-4094-8e2e-9e7b3a662090

Image:
SCR-20241227-skqk-2

Moved also the implementation of the language picker to the same style
SCR-20241227-skrb

Additionally:
- removed some unused files 
- added translations

Pending adjustments: 
- [x] add theme configuration
- [x] improve mobile spacing
- [x] add upgrading steps (as I combined some CSS files)
- [x] review impact on theme size/performance
– potential followup: load only some bootstrap files instead of the whole thing: https://getbootstrap.com/docs/5.3/customize/sass/#variable-defaults 
- [x] feedback
- [ ] correctly display the selected theme (☀️ / 🌑 / 🪄 ) [🐛 currently it always displays ☀️]

### CODE REVIEW

**Comment by dcorto** [comment]
Looks awesome @zetxek :)

**Comment by zetxek** [comment]
In terms of performance it seems there's no impact. We're good:

Mobile:
SCR-20241227-tgdj-2

Desktop:
SCR-20241227-tggo

The SEO decline is just because we're using the preview URLs in vercel to test (https://adritian-demo-git-theme-update-update-theme-7a59da-adrianmoreno.vercel.app/?vercelToolbarCode=A1_o93MWTpyHAW0#, from https://github.com/zetxek/adritian-demo/pull/115)

So it seems we're quite good. I'll test a bit more in mobile tomorrow (in a real device), and if it goes well we can ship this to a new release :)

File: i18n/en.yaml
Comment (dcorto) [inline i18n/en.yaml:1]: @zetxek maybe is worth it mention the new keys in the i18n files on `upgrading.md` file

File: upgrading.md [hunk 1]
```diff
@@ -2,6 +2,77 @@
 
 This documentation is meant to help you upgrade across versions, when potentially breaking changes are introduced.
 
+## v1.4.6
```
Comment (dcorto) [inline upgrading.md]: this will generate the v1.4.8 right?

**Comment by dcorto** [comment]
I have tested in my local site and works great, good job! I left some comments in the PR in specific files just for let you know ;)

Great work @zetxek

**Review by dcorto — APPROVED** [review]

**Comment by zetxek** [comment]
> I have tested in my local site and works great, good job! I left some comments in the PR in specific files just for let you know ;)
> 
> 
> 
> Great work @zetxek 

I love the comments! Ideally the theme is both usable "as-is" and extensible for other usages - many times when I'm implementing things i think "would anyone else use it like this?". So the feedback is super appreciated 🙌

File: upgrading.md (hunk 1 above)
Comment (zetxek) [reply upgrading.md]: you're so right! 😄

Comment (zetxek) [reply upgrading.md]: Corrected in 7140b6ec3557a1878ae4541521e15cb80daa95ae :)

File: i18n/en.yaml
Comment (zetxek) [reply i18n/en.yaml:1]: Good idea!

Comment (zetxek) [reply i18n/en.yaml:1]: Added in 37dad4669d796dfb138399e26d9443e4f4f87393

**Comment by zetxek** [comment]
I found a bug: the icon for the theme switcher should switch to the active one. It stays always in the ☀️ one.

**Comment by dcorto** [comment]
> I found a bug: the icon for the theme switcher should switch to the active one. It stays always in the ☀️ one.
> 
>  [  ](https://private-user-images.githubusercontent.com/240085/399081214-398afefe-5b6f-44c5-98ab-27472185f385.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzOTMxMjYsIm5iZiI6MTczNTM5MjgyNiwicGF0aCI6Ii8yNDAwODUvMzk5MDgxMjE0LTM5OGFmZWZlLTViNmYtNDRjNS05OGFiLTI3NDcyMTg1ZjM4NS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyOFQxMzMzNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNDlhM2JhYjBiYmMxODQ4MjdhMWJlN2IyMTFlYTgxMTc3NzFjZWE3ODIxYTdjN2UzMTk4ODgwNGU3OWY2YmNkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xmXjcRSZNOftIZvP8blnUlpYcGNyBjMy8XtQf3xc8oY) [ ](https://private-user-images.githubusercontent.com/240085/399081214-398afefe-5b6f-44c5-98ab-27472185f385.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzOTMxMjYsIm5iZiI6MTczNTM5MjgyNiwicGF0aCI6Ii8yNDAwODUvMzk5MDgxMjE0LTM5OGFmZWZlLTViNmYtNDRjNS05OGFiLTI3NDcyMTg1ZjM4NS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyOFQxMzMzNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNDlhM2JhYjBiYmMxODQ4MjdhMWJlN2IyMTFlYTgxMTc3NzFjZWE3ODIxYTdjN2UzMTk4ODgwNGU3OWY2YmNkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xmXjcRSZNOftIZvP8blnUlpYcGNyBjMy8XtQf3xc8oY)

Ah, of course. When I tested it, it seemed to be working as it should.

**Comment by zetxek** [comment]
I got this one right now :) 



It switches properly, it changes color if the theme is changed, and the language and theme selectors are consistent in styling.

**Comment by Aaron-Mok** [comment]
This is awesome! Thank you!

**Pull request merged**